src/compression/move_to_front.rs

```diff
@@ -0,0 +1,56 @@
+// https://en.wikipedia.org/wiki/Move-to-front_transform
+
+pub fn move_to_front_encode(text: &str) -> Vec<u8> {
+    dbg!(text);
+    let mut char_table: Vec<char> = (0..=255).map(|ch| ch as u8 as char).collect();
```

Comment:
What do you think about introducing a function like:
```rust
fn blank_char_table() -> Vec<char> {
    (0..=255).map(|ch| ch as u8 as char).collect()
}
```
and then
```suggestion
    let mut char_table: Vec<char> = blank_char_table();
```
and same in line 20.

Reply:
changed the two